File: 1115.py
y=x=1


while x != 0 and y != 0:
    x, y = map(int, input().split())
    if x > 0:
        if y > 0:
            print('primeiro')
        if y < 0:
            print('quarto')
    if x < 0:
        if y > 0:
            print('segundo')
        if y < 0:
            print('terceiro')
    
        

File: CSharp/1115.cs
using System;
using System.Linq;
class URI
{
    static void Main(string[] args)
    {
        int[] x;
        while(true)
        {
            x = Console.ReadLine().Split().Select(int.Parse).ToArray();
            if(x[0]>0 && x[1]>0)
            {
                Console.WriteLine("primeiro");
            }
            else if(x[0]>0 && x[1]<0)
            {
                Console.WriteLine("quarto");
            }
            else if(x[0]<0 && x[1]>0)
            {Console.WriteLine("segundo");}
            else if(x[0]<0 && x[1]<0)
            {Console.WriteLine("terceiro");}
            else if(x[0]==0 || x[1]==0)
            {break;}
        }
    }
}
